modules/access_affinitygroup/access_affinitygroup.module

```diff
@@ -613,12 +613,21 @@ function access_affinitygroup_cron() {
     // Cronrunning true means allocations cron Todo, rename.
     \Drupal::state()->set('access_affinitygroup.cronrunning', FALSE);
 
+    // Make sure we are not somehow on the support domain, This is crucial for
+    // the constant contact connection.
+    $host = \Drupal::request()->getHttpHost();
```

Comment:
@FloQuinn did you test this? I am having trouble finding docs describing this and don't have an easy way to test.

Reply:
@a-pasquale I force-simulated it,   and looked at the log to see $host, but I can't actually prove the positive without being on the site